app/views/users/show.html.erb

```diff
@@ -1,2 +1,2 @@
-<h1>Here your user with id</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<%= render "userid" %>
```

Comment:
- [ ] kindly fix the image in the user show page
![Screenshot from 2023-05-20 07-21-35](https://github.com/Becky449/Myblog/assets/66097371/7a352b22-a0c5-4621-a8ff-eeab73bc59b3)